lib/backend/sanitize_test.go

```diff
@@ -41,6 +41,54 @@ func TestSanitize(t *testing.T) {
 			inKey:  []byte("/namespaces//params"),
 			assert: require.Error,
 		},
+		{
+			inKey:  []byte("/namespaces/.."),
+			assert: require.Error,
+		},
+		{
+			inKey:  []byte("/namespaces/../params"),
+			assert: require.Error,
+		},
+		{
+			inKey:  []byte("/namespaces/..params"),
+			assert: require.NoError,
+		},
+		{
+			inKey:  []byte("/namespaces/."),
+			assert: require.Error,
+		},
+		{
+			inKey:  []byte("/namespaces/./params"),
+			assert: require.Error,
+		},
+		{
+			inKey:  []byte("/namespaces/.params"),
+			assert: require.NoError,
+		},
+		{
+			inKey:  []byte(".."),
+			assert: require.Error,
+		},
+		{
+			inKey:  []byte("..params"),
+			assert: require.NoError,
+		},
+		{
+			inKey:  []byte("../params"),
+			assert: require.Error,
+		},
+		{
+			inKey:  []byte("."),
+			assert: require.Error,
+		},
+		{
+			inKey:  []byte(".params"),
+			assert: require.NoError,
+		},
+		{
+			inKey:  []byte("./params"),
+			assert: require.Error,
+		},
```

Comment:
Test for `//` ?

Reply:
Nevermind, that was already in the list.

Reply:
That test is already present, fortunately!